build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.9.+'
+        classpath 'com.android.tools.build:gradle:1.2.3'
```

Comment:
Ah, okay... So you've added it as a git submodule -- but in addition to that you actually have to include the project in Gradle as a module!